src/serial_device.cpp

```
@@ -273,6 +269,27 @@ void TSerialDevice::SetLastReadTime(std::chrono::steady_clock::time_point readTi
     LastReadTime = readTime;
 }
 
+void TSerialDevice::AddOnConnectionStateChangedCallback(TSerialDevice::TDeviceCallback callback)
```

cppcheck

[style] The function 'AddOnConnectionStateChangedCallback' is never used.

(unusedFunction)